Sources/Network/TaxonomiesService.swift

```diff
@@ -31,7 +33,12 @@ enum TaxonomiesRouter: URLRequestConvertible {
     // MARK: URLRequestConvertible
 
     func asURLRequest() throws -> URLRequest {
-        let urlStr = Endpoint.get + "/data/taxonomies/" + self.path
+        var urlStr = ""
+        if self != .getIngredientsAnalysisConfig {
```

Comment:
This is a weird solution adopted here. I would suggest creating another router like `FilesRouter` with (for now) only the `.getIngredientsAnalysisConfig`. This way the asUrlRequest is simpler.